src/validate-markup.ts

```diff
@@ -17,6 +19,12 @@ export default async function main({ dest, file }: Input) {
 	await install("vnu-jar");
 	const vnuJar = require("vnu-jar");
 
+	const matchersPath = path.join(
+		__dirname,
+		"validate-markup-problem-matcher.json",
+	);
+	core.info(`##[add-matcher]${matchersPath}`);
```

Comment:
I was hoping there to be a `core.addMatcher(filePath)`, but it doesn't exist. Let's keep it in action.yml then.

Reply:
Looking at those other setup-* Actions, I think you were right around the path relative to the action, so this might be better (and more cross platform)